### PR TITLE
Nmr 7007 use constants for acquisitions types

### PR DESCRIPTION
**Pull request by vfiack**

### CODE REVIEW

File: nmrfx-analyst/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java [hunk 1]
```diff
@@ -1084,36 +1085,31 @@ private void setFTpars() {
...
                         fttype[i - 1] = "negate";
-                        f1coefS[i - 1] = "hyper";
+                        f1coefS[i - 1] = AcquisitionType.HYPER.getLabel();
                         break;
                     case 6:
-                        f1coef[i - 1] = new double[]{1, 0, -1, 0, 0, 1, 0, 1};
+                        f1coef[i - 1] = AcquisitionType.ECHO_ANTIECHO.getCoefficients();
                         deltaPh0_2 = 90.0;
-                        f1coefS[i - 1] = "echo-antiecho-r";
+                        f1coefS[i - 1] = AcquisitionType.ECHO_ANTI_ECHO_R.getLabel();
```
[the remark this answers is not in the document]
Comment (JannalieT) [reply nmrfx-analyst/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java:1104]: There were a couple other places with similar strangeness so I tried to comment them so we don't miss it

[the remark this answers is not in the document]
Comment (vfiack) [reply nmrfx-analyst/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java:1104]: Thanks. I had missed the others. We'll wait a bit for Bruce to reply, but maybe we'll need revert the coefficients and only use constants for names then 😞

[the remark this answers is not in the document]
Comment (brucejohnson) [reply nmrfx-analyst/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java:1104]: There was definitely a bug where the list of coefficients in the Bruker code didn't match the name.  The names take priority in normal use, so the coefficients that are set in the BrukerData file should be the ones that match the name.  
so if
       f1coefS[i-1] = AcquisitionType.ECHO_ANTI_ECHO_R.getLabel() 
then
        f1coef[i-1] = AcquisitionType.ECHO_ANTI_ECHO_R.getCoefficients()

**Comment by vfiack** [comment]
After discussions, let's assume that the symbolic names used were correct for Jcamp & spinit data, and that the coefficients were wrong. So I propose to merge this PR and change both coefficient + label if we notice a bug with some acquisition types, because they should be consistent.

**Review by JannalieT — APPROVED** [review]

**Pull request merged**